Dockerfile.redhat

```diff
@@ -195,11 +249,24 @@ ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/intel/openvino/runtime/lib/intel64/:
 RUN wget -nv https://github.com/linux-test-project/lcov/releases/download/v1.16/lcov-1.16-1.noarch.rpm &&\
     yum install -y lcov-1.16-1.noarch.rpm && yum clean all
 
-COPY check_coverage.bat /ovms/
+# Copy build dependencies
+COPY --from=base_build /awssdk/build/.deps /awssdk/build/.deps
```

Comment:
It might cleaner to just clean the aws installation in base image and just copy entire artifacts in single `COPY --from` call

Reply:
The multiple `COPY --from` steps are duplication what `make install` should do after aws installtion

Reply:
The same goes for ubuntu